build.py

```diff
@@ -1708,6 +1708,30 @@ def tensorrtllm_prebuild(cmake_script):
     # Export the TRT_ROOT environment variable
     cmake_script.cmd("export TRT_ROOT=/usr/local/tensorrt")
     cmake_script.cmd("export ARCH=$(uname -m)")
+    cmake_script.cmd(
+        'export LD_LIBRARY_PATH="/usr/local/cuda/compat/lib.real:${LD_LIBRARY_PATH}"'
```

Comment:
Why is this line required?

Reply:
Because `libcuda.so.1` can't be resolved till `--runtime=nvidia` is passed. 
It's a workaround that allows us refer to the links in build container, can be deprecated in future. 

You can create task and assign to me.